scripts/serialize_convention.py

```diff
@@ -151,23 +141,21 @@ def write_json_schema(filename, object):
         jsonfile.write(object)
 
 
-def generate_output_name(inputname, label, path=''):
+def check_if_old_output(project, version):
```

Comment:
since this doesn't just return a true/false check, consider renaming to get_input_output_names or something similar

Reply:
addressed in `41272306`